src/lsp-server.ts

```diff
@@ -588,7 +588,9 @@ export class LspServer {
             const result = await this.interuptDiagnostics(() => this.tspClient.request(CommandTypes.CompletionInfo, {
                 file,
                 line: params.position.line + 1,
-                offset: params.position.character + 1
+                offset: params.position.character + 1,
+                triggerCharacter: getCompletionTriggerCharacter(params.context?.triggerCharacter),
```

Comment:
This part doesn't really seem relevant to fixing the issue but it's good to have nonetheless.